app/controllers/users_controller.rb

```diff
@@ -1912,6 +1913,11 @@ def user_menu_messages
            }
   end
 
+  def lookup
+    users = User.where(username: params[:usernames])
```

Comment:
Is there any way this could be abused to list users that someone else shouldn't be able to see?

Any rate limits?

Reply:
We go with default rate limits here, also this method is available only to authorized users, which is I believe what we do with this kind of methods by default. 

> No limits on the number of usernames passed by the user?

I've limited it to 20 users in 0233046df11ed7ff1be15098377338f9b8f678b1.